app/src/main/java/com/imashnake/animite/features/searchbar/SearchBar.kt

```diff
@@ -76,8 +77,10 @@ fun SearchBar(
                         // TODO: Double clicking makes the navigation happen twice.
                         navController.navigate(
                             MediaPageDestination(
-                                id = it,
-                                mediaTypeArg = MediaType.ANIME.rawValue
+                                MediaPageArgs(
+                                    it!!,
```

Comment:
sus

Reply:
It's not possible for the user to select an item with no ID, I did not change SearchList to keep the change count down